### ISSUE TITLE
Backend Django issues

### ISSUE THREAD

**Issue by to-sta** [activist-org/activist#348]
### Terms

- [X] I have searched all [open bug reports](https://github.com/activist-org/activist/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

Currently if we run the django server it won't start. The errors can be seen below:
Fixing the errors is a prerequisite for testing other backend tasks #329.

```
Watching for file changes with StatReloader
Performing system checks...

Exception in thread django-main-thread:
Traceback (most recent call last):
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\tobia\Documents\Projects\contrib\activist\activist\venv\lib\site-packages\django\utils\autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\tobia\Documents\Projects\contrib\activist\activist\venv\lib\site-packages\django\core\management\commands\runserver.py", line 133, in inner_run
    self.check(display_num_errors=True)
  File "C:\Users\tobia\Documents\Projects\contrib\activist\activist\venv\lib\site-packages\django\core\management\base.py", line 556, in check
    raise SystemCheckError(msg)
django.core.management.base.SystemCheckError: SystemCheckError: System check identified some issues:

ERRORS:
<class 'entities.admin.OrganizationAdmin'>: (admin.E108) The value of 'list_display[2]' refers to 'application_id', which is not a callable, an attribute of 'OrganizationAdmin', or an attribute or method on 'entities.Organization'.
authentication.Support.supported_type: (fields.E304) Reverse accessor 'SupportEntityType.support_set' for 'authentication.Support.supported_type' clashes with reverse accessor for 'authentication.Support.supporter_type'.
        HINT: Add or change a related_name argument to the definition for 'authentication.Support.supported_type' or 'authentication.Support.supporter_type'.
authentication.Support.supporter_type: (fields.E304) Reverse accessor 'SupportEntityType.support_set' for 'authentication.Support.supporter_type' clashes with reverse accessor for 'authentication.Support.supported_type'.
        HINT: Add or change a related_name argument to the definition for 'authentication.Support.supporter_type' or 'authentication.Support.supported_type'.

WARNINGS:
?: (urls.W002) Your URL pattern '/v1/organizations/' has a route beginning with a '/'. Remove this slash as it is unnecessary. If this pattern is targeted in an include(), ensure the include() pattern has a trailing '/'.
?: (urls.W002) Your URL pattern '/v1/organizations/<int:organization_id>' has a route beginning with a '/'. Remove this slash as it is unnecessary. If this pattern is targeted in an include(), ensure the include() pattern has a trailing '/'.

System check identified 5 issues (0 silenced).
```

These are mostly minor changes like, deleting duplicate fields:

```
class Support(models.Model):
    supporter_type = models.ForeignKey("SupportEntityType", on_delete=models.CASCADE)
    supporter_entity = models.IntegerField(null=True)
    supported_type = models.ForeignKey("SupportEntityType", on_delete=models.CASCADE)
    supported_entity = models.IntegerField(null=True)

    def __str__(self):
        return self.id
```
I already made a fix for the issues and can submit a PR 😃

**Comment by andrewtavis** [comment]
Feel free to send along a PR @to-sta and we’ll chat in there! :) Thanks for brining this up and working on it already 😊

**Comment by andrewtavis** [comment]
Closed via #351 😊 Thank you, @to-sta!
